incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -14,6 +15,13 @@ import (
 )
 
 var _ = Describe("HNCConfiguration", func() {
+	// sleepSec is the time (in seconds) to sleep for objects propagation to take effect.
+	// time.Sleep only accepts integer values as input and from experiment 1
+	// second is the minimal integer seconds HNC needs to propagate an object.
+	// Therefore, we set sleep time as double of the minimal sleep time.
+	//
+	// We may need to increase the sleep time in future if HNC takes longer to propagate objects.
+	const sleepSec = 2
```

Comment:
Ah - this isn't how `time` works in golang :) See: https://golang.org/pkg/time/#Sleep:

```
time.Sleep(100 * time.Millisecond)
```

You've created an untyped constant here, which can't be passed directly to `time.Sleep()`. If you said:

```
const sleepSec = 2 * time.Second
```

then you wouldn't need to pass the units in the call to `time.Sleep`. Better yet, you could experiment with, say, `100 * time.Millisecond` to see if shorter times are still applicable.

Do you want to try out shorter times in this PR or another one?

Reply:
Thanks so much for the explanation. I tried the millisecond. Looks like the minimal time it needs is around 15 millisecond (0.015s). As we discussed, I changed sleep time to 1 second.